consensus/src/chained_bft/liveness/proposal_generator.rs

```diff
@@ -77,6 +77,19 @@ impl<T: Payload> ProposalGenerator<T> {
         Ok(Block::new_nil(round, hqc.as_ref().clone()))
     }
 
+    /// Reconfiguration rule - we propose empty blocks with parents' timestamp
+    /// after reconfiguration until it's committed
+    pub fn generate_reconfig_suffix(&self, round: Round) -> failure::Result<BlockData<T>> {
```

Comment:
can we rename this into generate_reconfig_nil / empty? Suffix makes sense after a few moments of thought but doesn't feel intuitive.